pkg/policy/l4.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 type AuxRule struct {
-	Expr string `json:"expr"`
+	Expr   string `json:"expr"`
+	L7type string `json:"L7type"`
```

Comment:
I don't think `type` is the right term here. Either use the same term as in the L4 filter (`L7Parser`) or `L7Proto`.

Reply:
done